src/app/dialogs/send-tokens/send-tokens-dialog.component.ts

```diff
@@ -82,7 +82,7 @@ export class SendTokensDialogComponent implements OnInit, OnDestroy {
                 type: TransactionType.TransferTokens,
                 from: this.appService.getAddressFromPrivateKey(this.formGroup.get('privateKey').value),
                 to: this.formGroup.get('to').value,
-                value: parseInt(this.formGroup.get('tokens').value, 10)
+                value: this.formGroup.get('tokens').value + ''
```

Comment:
Is the + '' for the case that tokens is null? Feels a little weird.

Reply:
@brianosaurus Type cast? Just a guess.